index.html

```diff
@@ -3858,6 +3865,17 @@ <h3>
             <li>Initialize <var>event</var>.<a>validationURL</a> attribute to
             <var>validationURL</var>.
             </li>
+            <li>If <var>eventInitDict</var>["<a>methodName</a>"] is not the
+            empty string, run the steps to <a data-cite=
+            "payment-method-id#dfn-validate-a-payment-method-identifier">validate
+            a payment method identifier</a> with
+            <var>eventInitDict</var>["<a>methodName</a>"]. If it returns false,
```

Comment:
I'm always unsure how much validation we should be doing in event constructors, i.e. should they be dumb property bags that hold whatever you pass in, or should they enforce domain logic constraints. But this is good for now, and consistent.

Reply:
Yeah, thought about the same. Consistency allows us to treat both implementers and user-land code as potentially invalid, which I think is a good thing.